### ISSUE TITLE
Adhere to the user's accent color

### ISSUE THREAD

**Issue by sindresorhus** [wulkano/kap#661]
Electron 5 has an API for this now: https://electronjs.org/blog/electron-5-0#system-colors-access-for-macos

**Comment by skllcrn** [comment]
We can include this issue in #644 @karaggeorge?

**Comment by sindresorhus** [comment]
@skllcrn I don't think we should add more tasks to that PR or v3. We can ship a v3.1 with additional stuff.

**Comment by skllcrn** [comment]
I’m in favour of it, we need to create a system for colours across modes that work perfectly and don't require high design or development effort, while still testing every single view with different combinations.
